### PR TITLE
New Post: How to Use Proprietary Foundation Models with Amazon SageMaker JumpStart for Building Generative AI Applications

### PR DESCRIPTION
**Pull request by channy**
…ker JumpStart for Building Generative AI Applications

Amazon SageMaker JumpStart supports popular proprietary foundation models, such as AI21 Labs, Cohere, and LightOn. In this blog post, learn how to use a preview feature, a playground of AWS console and deploy to SageMaker Studio. You can test these models without writing a single line of code and easily evaluate models on your specific tasks.

# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [x] The title of this pull request reflects the title of my content.
- [x] I have reviewed the [Content Review Checklist](/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [x] This pull request contains one piece of content (post or page).

**Important: if you are submitting a PR to update existing content, please ensure you [sync'ed your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) first before you created your branch.**

Is this pull request linked to a Content Proposal or Idea Suggestion Issue? If so, please link to it below using the number or full link.

```text
Resolves #5

or

Content for https://github.com/build-on-aws/content/issues/5
```

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

**Comment by mikegc-aws** [comment]
Thanks for this post.  Here are some review feedback notes: 

- Please can you add your images into an ./images folder and reference them from there.
- [25] "you can simply create an image on the right by feeding the sentence" It does not show in the right in my view.  Its underneath.  Should we maybe just reference the image more generally? 
- [81] Code - Does that code run?  Looks like the ARN has a dummy account number in it.  Correct me if I'm wrong, but is that not supposed to be an account number for the SageMaker service account, and as such fine to be public.  In any case can we clarify if this is 'example code' that can't be run as is. 
- [81] Code - The sample code will spin up a `g5.xlarge` instance, which runs around $1 an hour.  Can we advise readers to investigate the costs, and include instructions on how to clean up afterwards and remove the endpoint.

File: posts/how-to-use-proprietary-foundation-models-with-amazon-sagemaker-jumpstart-for-building-generative-ai-applications/index.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+![](https://d2908q01vomqb2.cloudfront.net/da4b9237bacccdf19c0760cab7aec4a8359010b0/2023/03/16/2023-generative-ai-sagemaker-1.jpg)
```
Comment (cobusbernard) [inline posts/how-to-use-proprietary-foundation-models-with-amazon-sagemaker-jumpstart-for-building-generative-ai-applications/index.md:21]: Can you please add alt-text in the `[]` please for all images @channy ?

File: posts/how-to-use-proprietary-foundation-models-with-amazon-sagemaker-jumpstart-for-building-generative-ai-applications/index.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+---
+title: "How to Use Proprietary Foundation Models with Amazon SageMaker JumpStart for Building Generative AI Applications"
+description: "Amazon SageMaker JumpStart supports popular proprietary foundation models, such as AI21 Labs, Cohere, and LightOn. In this blog post, learn how to use a preview feature, a playground of AWS console and deploy to SageMaker Studio. You can test these models without writing a single line of code and easily evaluate models on your specific tasks."
+tags:
+  - ai-ml
+  - generative-ai
```
Comment (cobusbernard) [inline posts/how-to-use-proprietary-foundation-models-with-amazon-sagemaker-jumpstart-for-building-generative-ai-applications/index.md:6]:
```suggestion
  - gen-ai
```

File: posts/how-to-use-proprietary-foundation-models-with-amazon-sagemaker-jumpstart-for-building-generative-ai-applications/index.md (hunk 1 above)
Comment (cobusbernard) [reply posts/how-to-use-proprietary-foundation-models-with-amazon-sagemaker-jumpstart-for-building-generative-ai-applications/index.md:21]: Also, can you please add the images to the `images` folder of you post?

File: posts/how-to-use-proprietary-foundation-models-with-amazon-sagemaker-jumpstart-for-building-generative-ai-applications/index.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+You can also choose a [broad range of AWS services](https://aws.amazon.com/blogs/machine-learning/announcing-new-tools-for-building-with-generative-ai-on-aws/) that have generative AI built in, all running on the most cost-effective cloud infrastructure for generative AI. To learn more, visit [Generative AI](https://aws.amazon.com/generative-ai/) on AWS to innovate faster to reinvent your applications.
+
+## About Author
+Channy Yun is a Principal Developer Advocate for AWS, and passionate about helping developers to build modern applications on latest AWS services. A pragmatic developer and blogger at heart, he loves community-driven learning and sharing of technology, which has funneled developers to global AWS Usergroups. His main topics are open-source, container, storage, network & security, and IoT. Follow him on Twitter at @channyun.
```
Comment (cobusbernard) [inline posts/how-to-use-proprietary-foundation-models-with-amazon-sagemaker-jumpstart-for-building-generative-ai-applications/index.md:116]: Can you change your twitter handle to be a link to your profile on twitter please?

File: posts/how-to-use-proprietary-foundation-models-with-amazon-sagemaker-jumpstart-for-building-generative-ai-applications/index.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    # If the endpoint is already created, you just need to connect to it
+    # co.connect_to_endpoint(endpoint_name="cohere-gpt-medium")
+
+    # Perform real-time inference
+    prompt="""Write a creative product description for a wireless headphone product named the CO-1T, with the keywords "bluetooth", "wireless", "fast charging" for a software developer who works in noisy offices, and describe benefits of this product."""
+    response = co.generate(prompt=prompt, max_tokens=100, temperature=0.9)
+    print(response.generations[0].text)
+```
+
+For a variety of tutorials and tips and tricks related to this language model, see [Cohere brings language AI to Amazon SageMaker](https://aws.amazon.com/ko/blogs/machine-learning/cohere-brings-language-ai-to-amazon-sagemaker/) in the AWS Machine Learning Blog. You can also see an [example of automated quiz generation with AWS Certification exams](https://aws.amazon.com/blogs/machine-learning/accelerate-your-learning-towards-aws-certification-exams-with-automated-quiz-generation-using-amazon-sagemaker-foundations-models/) using AI21 Jurassic-2 Jumbo Instruct model.
```
Comment (cobusbernard) [inline posts/how-to-use-proprietary-foundation-models-with-amazon-sagemaker-jumpstart-for-building-generative-ai-applications/index.md:106]:
```suggestion
For a variety of tutorials and tips and tricks related to this language model, see [Cohere brings language AI to Amazon SageMaker](https://aws.amazon.com/blogs/machine-learning/cohere-brings-language-ai-to-amazon-sagemaker/) in the AWS Machine Learning Blog. You can also see an [example of automated quiz generation with AWS Certification exams](https://aws.amazon.com/blogs/machine-learning/accelerate-your-learning-towards-aws-certification-exams-with-automated-quiz-generation-using-amazon-sagemaker-foundations-models/) using AI21 Jurassic-2 Jumbo Instruct model.
```

File: posts/how-to-use-proprietary-foundation-models-with-amazon-sagemaker-jumpstart-for-building-generative-ai-applications/index.md (hunk 2 above)
Comment (cobusbernard) [reply posts/how-to-use-proprietary-foundation-models-with-amazon-sagemaker-jumpstart-for-building-generative-ai-applications/index.md:6]: Can you also please add a tag for "sagemaker"?

**Review by cobusbernard — CHANGES_REQUESTED** [review]

**Comment by channy** [comment]
Thanks mikegc-aws and cobusbernard! Incorporated your feedback.

**Review by cobusbernard — APPROVED** [review]

**Review by cobusbernard — APPROVED** [review]

**Pull request merged**